lib/browser/api/web-contents.ts

```diff
@@ -543,6 +548,11 @@ WebContents.prototype._init = function () {
 
   this.on('crashed', (event, ...args) => {
     app.emit('renderer-process-crashed', event, this, ...args);
+
+    // Log out a hint to help users better debug renderer crashes.
+    if (loggingEnabled()) {
+      console.info('Renderer process crashed - see https://www.electronjs.org/docs/tutorial/application-debugging for potential debugging information.');
```

Comment:
Can we give some more info about the crash here?

Reply:
https://github.com/electron/electron/pull/25592